pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -787,6 +787,15 @@ type (
 	}
 )
 
+// MetricsCollection used by the scaling config
+// https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-metricscollection.html
+type MetricsCollection struct {
+	// +required
+	Granularity string `json:"granularity"`
+	// +optional
+	Metrics []string `json:"metrics"`
```

Comment:
As this one is optional field, do you want to add omitempty tag as well ? Otherwise, this attiribute is showing as `required` in schema file.